### PR TITLE
dev-lang/mono-basic: bump to version 4.8

### PR DESCRIPTION
**Pull request by ConiKost**
Package-Manager: Portage-2.3.64, Repoman-2.3.12
Signed-off-by: Conrad Kostecki <conrad@kostecki.com>

### CODE REVIEW

File: dev-lang/mono-basic/mono-basic-4.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit mono-env
+
+MY_PV="e31cb702937a0adcc853250a0989c5f43565f9b8"
```
Comment (mgorny) [inline dev-lang/mono-basic/mono-basic-4.8.ebuild]: Suggested name: EGIT_COMMIT.

File: dev-lang/mono-basic/mono-basic-4.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+DESCRIPTION="Visual Basic Compiler and Runtime"
+HOMEPAGE="https://www.mono-project.com/docs/about-mono/languages/visualbasic/"
+SRC_URI="https://github.com/mono/${PN}/archive/e31cb702937a0adcc853250a0989c5f43565f9b8.tar.gz -> ${P}.tar.gz"
```
Comment (mgorny) [inline dev-lang/mono-basic/mono-basic-4.8.ebuild]: I suppose you want to reuse the variable here ;-P.

File: dev-lang/mono-basic/mono-basic-4.8.ebuild [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="LGPL-2 MIT"
+SLOT="0"
+
+RDEPEND="dev-lang/mono"
+DEPEND="${RDEPEND}"
+
+RESTRICT="test"
```
Comment (mgorny) [inline dev-lang/mono-basic/mono-basic-4.8.ebuild]: Why oh why?

Comment (ConiKost) [reply dev-lang/mono-basic/mono-basic-4.8.ebuild]: Will add a comment. There are multiple test failures, which seems to be caused by not updated for current mono: https://github.com/mono/mono-basic/issues/49 Upstream hasn't replied.

File: dev-lang/mono-basic/mono-basic-4.8.ebuild (hunk 2 above)
Comment (ConiKost) [reply dev-lang/mono-basic/mono-basic-4.8.ebuild]: This is pro style xP, changed.

File: dev-lang/mono-basic/mono-basic-4.8.ebuild (hunk 1 above)
Comment (ConiKost) [reply dev-lang/mono-basic/mono-basic-4.8.ebuild]: Okay, fine for me.

**Comment by mgorny** [comment]
Thanks, I think it's good now.

**Pull request closed without merging**